Resources/Locale/en-US/components/kitchen-spike-component.ftl

```diff
@@ -1,16 +1,16 @@
 comp-kitchen-spike-deny-collect = { THE($this) } already has something on it, finish collecting its meat first!
-comp-kitchen-spike-deny-butcher = { THE($victim) } can't be butchered on { THE($this) }.
-comp-kitchen-spike-deny-not-dead = { THE($victim) } can't be butchered. { THE($victim) } is not dead!.
+comp-kitchen-spike-deny-butcher = { $victim } can't be butchered on { $this }.
```

Comment:
shouldn't it be
```suggestion
comp-kitchen-spike-deny-butcher = { $victim } can't be butchered on { THE($this) }.
```
?

Reply:
There are additional concerns.
Context: I wrote the first iteration of this component (before it got Fluented).
In that version, the equivalent of "THE" here did not *always* add a "the", but instead would write either, say, "The monkey can't be butchered." vs. "20kdc can't be butchered."
I'm not sure how that is preserved here.

Reply:
The point of having `THE($thing)` is so that proper nouns don't get `the` added to them.
the monkey can't be butchered on the meat spike.
John Doe can't be butchered on the meat spike.
John Doe can't be butchered on Spiky.

Humans are just missing the grammar component and proper noun tag.
```
  - type: Grammar
    attributes:
      proper: true
```